apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -8,42 +8,44 @@
             <translate>Add Collaborators</translate>
           </oc-button>
         </div>
-        <translate
-          v-else
-          class="files-collaborators-no-reshare-permissions-message"
-          key="no-reshare-permissions-message"
-          v-text="noResharePermsMessage"
-        />
+        <p v-else>
```

Comment:
If the change is only because of having the `p` element then we can use either `v-translate` on that element or specify `tag="p"` on translate element. Either way, we can drop the extra element then.

Reply:
Now that you point it out, I am wondering why there was a translate tag in the first place. The `v-text` brings in text which is already translated. I merged the `p` and `translate` into a single `p` tag without any `v-translate`. ;-) Thanks for finding this 💪

Reply:
Thanks! 👍 It escaped me that it was already translated 🙈